src/components/Practice/Mechanics4.jsx

```
@@ -148,80 +166,131 @@ const Mechanics4 = ({
                     : "#333F61",
-                fontWeight: type == "word" ? 600 : 700,
-                fontSize: "40px",
+                fontWeight: type === "word" ? 600 : 700,
+                fontSize: "clamp(1.5rem, 2.5vw, 2.5rem)",
                 fontFamily: "Quicksand",
                 cursor: "pointer",
-                marginLeft: type == "word" ? 0 : "20px",
+                marginLeft:
+                  type === "word" ? 0 : answer != "correct" ? "20px" : 0,
               }}
             >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)